inst/tests/tests.Rraw

```diff
@@ -7359,6 +7359,7 @@ test(1514.6, isRealReallyInt32(x), FALSE)
 x = numeric()
 test(1514.7, isRealReallyInt32(x), TRUE)
 test(1514.8, isRealReallyInt32(9L), FALSE) # b/c not double input
+test(1514.9, isRealReallyInt64(9L), FALSE)
```

Comment:
thanks! no harm keeping it now that you've already gone to the trouble, but for future reference these days I prefer not to test private API directly, especially not just for codecov sake.

Reply:
OK noted